log10/llm.py

```diff
@@ -144,6 +142,12 @@ def last_completion_url(self):
             + self.last_completion_response["completionID"]
         )
 
+    def get_duration(self):
```

Comment:
What do you think about "last_duration" for consistency with the "last_completion_url"?